openquakeplatform/bin/oq_install.sh

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env bash 
+# GeoNode installer script
+#
+# using getopts
+#
+
+if [ $GEM_SET_DEBUG ]; then
+     set -x
+fi
+set -e
+
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root" 1>&2
+    exit 1
+fi
+
+while getopts 's:' OPTION
+do
+    case $OPTION in
+    s)
+        stepflag=1
+        stepval="$OPTARG"
+        ;;
+    ?)
+        printf "Usage: %s: [-s value] configfile\n" $(basename $0) >&2
+        exit 2
+        ;;
+    esac
+done
+shift $(($OPTIND - 1))
+
+function unsudo () {
+    local normal_user cmd="$1"
+
+    normal_user="$(logname)"
+    sudo -u $normal_user -i bash -c "$cmd"
+}
+
+function setup_directories() {
+    mkdir -p $GEOSERVER_DATA_DIR
+    mkdir -p $GEONODE_WWW/static
+    mkdir -p $GEONODE_WWW/wsgi
+    mkdir -p $APACHE_SITES
+    mkdir -p $GEONODE_BIN
+    mkdir -p $GEONODE_ETC
+    mkdir -p $GEONODE_ETC/media
+    mkdir -p $GEONODE_ETC/templates
+    mkdir -p $GEONODE_SHARE
+
+    # Create an empty uploads dir
+    mkdir -p $GEONODE_WWW/uploaded
+    mkdir -p $GEONODE_WWW/uploaded/thumbs/
+    mkdir -p $GEONODE_WWW/uploaded/layers/
+    mkdir -p $GEONODE_WWW/uploaded/documents/
+
+    # Open up the permissions of the media folders so the python
+    # processes like updatelayers and collectstatic can write here
+    chmod 775 -R $GEONODE_WWW
+    chmod g+s $GEONODE_WWW/uploaded/thumbs
+    chmod g+s $GEONODE_WWW/static
+
+    # Apply the permissions to the newly created folders.
+    chown www-data.www-data -R $GEONODE_WWW
+}
+
+function reorganize_configuration() {
+    cp -rp $INSTALL_DIR/support/geonode.apache $APACHE_SITES/geonode.conf
+    cp -rp $INSTALL_DIR/support/geonode.wsgi $GEONODE_WWW/wsgi/
+    if [ "$DEVEL_DATA" ] || [ "$DATA_PROD" ]; then
```

Comment:
why not a more simple:

``    if [ "$DEVEL_DATA" -o "$DATA_PROD" ]; then ``
￼

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/66/commits/f22ef5606003e39798fe23beb39fb39c9bd5c7fc